services/skus/controllers_noint_test.go

```diff
@@ -222,3 +223,151 @@ func TestHandleReceiptErr(t *testing.T) {
 		})
 	}
 }
+
+func TestParseVerifyCredRequestV2(t *testing.T) {
+	type tcExpected struct {
+		val   *model.VerifyCredentialRequestV2
+		errFn must.ErrorAssertionFunc
+	}
+
+	type testCase struct {
+		name  string
+		given []byte
+		exp   tcExpected
+	}
+
+	tests := []testCase{
+		{
+			name:  "error_malformed_payload",
+			given: []byte(`nonsense`),
+			exp: tcExpected{
+				errFn: func(tt must.TestingT, err error, i ...interface{}) {
+					must.Equal(tt, true, err != nil)
+				},
+			},
+		},
+
+		{
+			name:  "error_malformed_credential",
+			given: []byte(`{"sku":"sku","merchantId":"merchantId"}`),
+			exp: tcExpected{
+				errFn: func(tt must.TestingT, err error, i ...interface{}) {
+					must.Equal(tt, true, err != nil)
```

Comment:
Same as above, would using `must.NotNil(tt, err)` not be more concise?

Reply:
Same reasoning as [here](https://github.com/brave-intl/bat-go/pull/2654#discussion_r1756217159).